src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/monitor/ProjectCreationSupervisor.scala

```diff
@@ -15,13 +15,28 @@ import org.broadinstitute.dsde.workbench.gpalloc.util.Throttler
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
+import scala.util.Random
 
 object ProjectCreationSupervisor {
+  var gpAllocConfig: GPAllocConfig = null
```

Comment:
this is hideous and i hate it. please help

Reply:
ProjectCreationSupervisor probably shouldn't be an object

Reply:
or parameterize `randomProjectName` below with the prefix and